### PR TITLE
Fix/unit tests

### PR DESCRIPTION
**Pull request by musamusa**

### CODE REVIEW

**Review by teehamaral — CHANGES_REQUESTED** [review]
There are two failed tests that need to be fixed:

```
--- FAIL: TestParseQuery (0.00s)
    parser_test.go:214: 
                Error Trace:    parser_test.go:214
                Error:          Error message not equal:
                                expected: "contains conditions can only be used with name or URN values"
                                actual  : "contains operator on field requires token of minimum length 3"
                Test:           TestParseQuery
                Messages:       error mismatch for 'gender ~ M'
    parser_test.go:214: 
                Error Trace:    parser_test.go:214
                Error:          An error is expected but got nil.
                Test:           TestParseQuery
                Messages:       error mismatch for 'gender ~ mal'
    parser_test.go:215: 
                Error Trace:    parser_test.go:215
                Error:          Expected nil, but got: &contactql.ContactQuery{root:(*contactql.Condition)(0xc000127040)}
                Test:           TestParseQuery
```

```
--- FAIL: TestGenerateDocs (1.27s)
    base_test.go:37: 
                Error Trace:    base_test.go:37
                Error:          Received unexpected error:
                                error event generated: webhook response body exceeds 10000 bytes limit
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.eventsForAction
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:448
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderActionDoc
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:293
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.createItemListContextFunc.func1
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:50
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.buildTemplateContext
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:236
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderTemplateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:83
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.(*htmlDocsGenerator).Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:65
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:110
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                                error running action
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderActionDoc
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:295
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.createItemListContextFunc.func1
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:50
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.buildTemplateContext
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:236
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderTemplateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:83
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.(*htmlDocsGenerator).Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:65
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:110
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                                error rendering action:call_shorten_url
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.createItemListContextFunc.func1
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html_renderers.go:51
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.buildTemplateContext
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:236
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderTemplateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:83
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.(*htmlDocsGenerator).Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:65
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:110
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                                error building docs context
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.renderTemplateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:85
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.(*htmlDocsGenerator).Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:65
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:110
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                                error rendering templates
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.(*htmlDocsGenerator).Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/html.go:66
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:110
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                                error invoking generator html docs
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.generateForLang
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:111
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs.Generate
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base.go:60
                                github.com/greatnonprofits-nfp/goflow/cmd/docgen/docs_test.TestGenerateDocs
                                        /Users/teehamaral/GNP/goflow/cmd/docgen/docs/base_test.go:36
                                testing.tRunner
                                        /usr/local/go/src/testing/testing.go:1108
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1374
                Test:           TestGenerateDocs
```

File: flows/events/base_test.go
Comment (musamusa) [reply flows/events/base_test.go:350]: weird, looks good from my IDE 
image

I'll compact it.

File: flows/actions/testdata/send_msg.json
Comment (musamusa) [reply flows/actions/testdata/send_msg.json:516]: I thought so, its just failing when return that value. I'll dig into the actual code to see why its not translating. should be a minor fix.

**Review by teehamaral — APPROVED** [review]

**Comment by teehamaral** [comment]
@musamusa - From my side, it's approved. Just fix the issue that Ihor caught and wait for his approval, then we can merge it. Thanks.

File: flows/actions/testdata/send_msg.json
Comment (musamusa) [reply flows/actions/testdata/send_msg.json]: @ihor-palii translation fixed

**Review by ihor-palii — APPROVED** [review]

**Comment by teehamaral** [comment]
Great. Thank you! @musamusa @ihor-palii

**Pull request merged**
